overlays/indexing-shortcut-rename.yaml

```diff
@@ -0,0 +1,21 @@
+overlay: 1.0.0
+x-speakeasy-jsonpath: rfc9535
+info:
+  title: Rename Indexing Shortcut component to avoid clash with client Shortcut
+  version: 0.0.5
+
+actions:
+  # 1. Duplicate the existing component under the new name
+  - target: "$.components.schemas"
+    update:
+      IndexingShortcut: "{{ $.components.schemas.Shortcut }}"
```

Comment:
To be honest with you I have never seen this syntax! 👏  Ideally you would update the source? Do you not have control over that then?